admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/structure/domain/StructuredContentImpl.java

```diff
@@ -73,13 +72,19 @@
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
 import javax.persistence.Transient;
+import javax.persistence.Index;
 
 /**
  * Created by bpolster.
  */
 @Entity
 @Inheritance(strategy = InheritanceType.JOINED)
-@Table(name = "BLC_SC")
+@Table(name = "BLC_SC", indexes = {
+        @Index(name="CONTENT_NAME_INDEX", columnList = "CONTENT_NAME , ARCHIVED_FLAG , SC_TYPE_ID"),
```

Comment:
Error: database column 'ARCHIVED_FLAG' not found;
Check columns and names in DB (there is an "ARCHIVED" column in BLC_SC)